### PR TITLE
feat: sell improvements

### PR DESCRIPTION
**Pull request by mohammed-ledger**
- arch improvement
-  add sig payload

### CODE REVIEW

File: pages/docs/exchange/sell/providers-backend.mdx [hunk 1]
```diff
@@ -21,11 +21,26 @@ There are 4 main endpoints needed for the sell:
...
+
 
 <details>
   <summary>Additional Details: POST /sell</summary>
-            
+
+  Example request payload:
+``` json copy
+  {
+  "quoteId": "123abc", // 123
```
Comment (sarneijim) [inline pages/docs/exchange/sell/providers-backend.mdx:33]:
```suggestion
  "quoteId": "123abc",
```

File: pages/docs/exchange/sell/providers-liveapp.mdx [hunk 2]
```diff
@@ -90,4 +79,4 @@ Your [LiveApp manifest](../../../../docs/discover/wallet-api/appendix/manifest)
 
 ## Flow/UX requirements
 
-Please refer to the [Flow Breakdown](../swap/flow) section for more information. 
\ No newline at end of file
+Please refer to the [Flow Breakdown](../sell/flow) section for more information.
```
Comment (pdesreux-ledger) [inline pages/docs/exchange/sell/providers-liveapp.mdx:82]: The [Flow Breakdown](https://developers.ledger.com/docs/exchange/swap/flow) section doesn't exist for sell, so this link will be broken. 
I don't remember who wrote it, but he only did for Swap. But I do believe lot of things are relevant for other operations tho (that's why we link it here anyways).

File: pages/docs/exchange/sell/providers-backend.mdx [hunk 3]
```diff
@@ -21,11 +21,26 @@ There are 4 main endpoints needed for the sell:
...
+  "amount": "string",
```
Comment (mstorozhenko-ledger) [inline pages/docs/exchange/sell/providers-backend.mdx:34]: amount should be a number

File: pages/docs/exchange/sell/providers-liveapp.mdx [hunk 4]
```diff
@@ -25,29 +25,18 @@ You will need to set up the SDK with your `providerId` attributed upon request b
...
-    getSellPayload: (nonce, sellAddress, amount) => {
-      // Crafting Payload...
-      return {
-        recipientAddress: "0xc0ffee254729296a45a3885639AC7E10F9d54979"
-        amount: BigInt("800000000000000000")
-        binaryPayload: "YiA1CuoMl_dH8dD3YIFGFKR1I4AbGut9C8u6oqT0a_gYSw==",
-        signature: "HA8qBfo8yFNzbV60Ws8GT54UWeing3kcDIdnov2nT9gFB58JybFE1sK0Ext3kARXhZ9J1sbltkBSeuzBdSkjVg==",
-      };
-    }
+  quoteId: "123abc", // 123
```
Comment (sarneijim) [inline pages/docs/exchange/sell/providers-liveapp.mdx:35]:
```suggestion
  quoteId: "SELL-ID-165940",
```

File: pages/docs/exchange/sell/providers-liveapp.mdx [hunk 5]
```diff
@@ -25,29 +25,18 @@ You will need to set up the SDK with your `providerId` attributed upon request b
...
+  fromAccountId: "97f06be9-6fb2-5da3-be71-4e762ed6e115",// Ledger Live uuid4 account (can be a BTC account)
+  fromAmount: new BigNumber(amount), // 1 
+  tofiat: "EUR", // EUR
```
Comment (sarneijim) [inline pages/docs/exchange/sell/providers-liveapp.mdx:38]: In swap the analog attribute would be `toAccountId`.
I wonder if we must use provider fiat Id, and we will need to convert in in the backend side. @mstorozhenko-ledger , what do you think? (we will be applying hte same logic for sell and for card)

```suggestion
  tofiat: "EUR", // provider fiat id
```

File: pages/docs/exchange/sell/providers-liveapp.mdx [hunk 6]
```diff
@@ -25,29 +25,18 @@ You will need to set up the SDK with your `providerId` attributed upon request b
...
+  rate: new BigNumber(amount),
```
Comment (sarneijim) [inline pages/docs/exchange/sell/providers-liveapp.mdx:39]:
```suggestion
  rate: 90000, // crypto/fiat rate [BTC/EUR]
```

File: pages/docs/exchange/sell/providers-liveapp.mdx [hunk 7]
```diff
@@ -25,29 +25,18 @@ You will need to set up the SDK with your `providerId` attributed upon request b
...
-      return {
-        recipientAddress: "0xc0ffee254729296a45a3885639AC7E10F9d54979"
-        amount: BigInt("800000000000000000")
-        binaryPayload: "YiA1CuoMl_dH8dD3YIFGFKR1I4AbGut9C8u6oqT0a_gYSw==",
-        signature: "HA8qBfo8yFNzbV60Ws8GT54UWeing3kcDIdnov2nT9gFB58JybFE1sK0Ext3kARXhZ9J1sbltkBSeuzBdSkjVg==",
-      };
-    }
+  quoteId: "123abc", // 123
+  fromAccountId: "97f06be9-6fb2-5da3-be71-4e762ed6e115",// Ledger Live uuid4 account (can be a BTC account)
+  fromAmount: new BigNumber(amount), // 1
```
Comment (sarneijim) [inline pages/docs/exchange/sell/providers-liveapp.mdx:37]:
```suggestion
  fromAmount: new BigNumber(1),
```

File: pages/docs/exchange/sell/providers-liveapp.mdx [hunk 8]
```diff
@@ -25,29 +25,18 @@ You will need to set up the SDK with your `providerId` attributed upon request b
...
-* `accountId`: user's account id, [previously given to the LiveApp](./providers-liveapp) as a query parameter
-* `amount`: amount that will be sold
-* `feeStrategy`: `SLOW`, `MEDIUM`, `FAST` or `CUSTOM`.
-* `getSellPayload`: callback method called by the sell process to retrieve the payload (see below)
-
-The `getSellPayload` is a callback that during the Sell process will be called with a `nonce`.
-This value is required to craft on your side the Sell Payload, which will be returned by the callback.
+* `fromAccountId`: user's account id, [previously given to the LiveApp](./providers-liveapp) as a query parameter
+* `fromAmount`: amount that will be sold
+* `tofiat`: fiat currency that the user wants to be paid with
```
Comment (sarneijim) [inline pages/docs/exchange/sell/providers-liveapp.mdx:30]: Need to confirm with @mstorozhenko-ledger 
```suggestion
* `tofiat`: fiat currency that the user wants to be paid with using provider id
```

File: pages/docs/exchange/sell/providers-backend.mdx [hunk 9]
```diff
@@ -21,11 +21,26 @@ There are 4 main endpoints needed for the sell:
...
+
 
 <details>
   <summary>Additional Details: POST /sell</summary>
-            
+
+  Example request payload:
+``` json copy
+  {
+  "quoteId": "123abc",
```
Comment (sarneijim) [inline pages/docs/exchange/sell/providers-backend.mdx:33]:
```suggestion
  "quoteId": "string",
```

**Review by sarneijim — APPROVED** [review]
Small typo

**Review by mstorozhenko-ledger — APPROVED** [review]

**Comment by mstorozhenko-ledger** [comment]
we will also need `/status` endpoint, similar to swap:
https://developers.ledger.com/docs/exchange/swap/providers-backend

**Pull request merged**